common/com/twitter/intellij/pants/util/PantsUtil.java

```diff
@@ -320,21 +320,34 @@ public static GeneralCommandLine defaultCommandLine(@NotNull File pantsExecutabl
       cmd.addParameters(
         "list",
         Paths.get(projectPath).getParent().toString() + ':',
-        String.format("%s=%s", PantsConstants.PANTS_CLI_OPTION_LIST_OUTPUT_FILE,
-                      tempFile.getFile().getPath()
-        )
+        String.format("%s=%s",
+                      PantsConstants.PANTS_CLI_OPTION_LIST_OUTPUT_FILE,
+                      tempFile.getFile().getPath())
       );
       final ProcessOutput processOutput = PantsUtil.getCmdOutput(cmd, null);
+      final String listOutput = FileUtil.loadFile(tempFile.getFile());
       if (processOutput.checkSuccess(LOG)) {
-        String output = FileUtil.loadFile(tempFile.getFile());
-        return Arrays.asList(output.split("\n"));
+        return Arrays.asList(listOutput.split("\n"));
       }
       else {
-        throw new PantsException("Failed:" + cmd.getCommandLineString());
+        final String errorDescription =
+          String.format("could not list targets: pants exited with status %d\n",
```

Comment:
nit: capitalization `Could not...... Pants ...`

Reply:
Fixed, also fixed in the other exception message to `"Could not execute command..."`.